chromium_src/check_chromium_src_config.json5

```diff
@@ -6,23 +6,18 @@
         ".*\\.clangd",
         ".*\\.gn",
         ".*\\.gni",
-        ".*\\.idl",
         ".*\\.in",
         ".*\\.info",
-        ".*\\.json",
-        ".*\\.json5",
-        ".*\\.mojom",
-        ".*\\.pdl",
-        ".*\\.py",
```

Comment:
@mkarolin I cleaned-up this list a bit to cover all files we override with "touch" logic during the build.

This way hopefully we can catch rebase issues at early stage.